javax-money/src/moditect/module-info.java

```diff
@@ -0,0 +1,13 @@
+//TODO how is this generated
```

Comment:
I am not sure how this file is generated.
Is there some documentation for this?

Reply:
I think it would be running

    ../mvnw moditect:generate-module-info

from new modules project dir. Let me try this on PR.

This provides the base, but must be modified; looking at other modules's `module-info.java` for inspiration.

Reply:
Hmmh. Alas, that command throws NPE.